ansible_collections/arista/avd/README.md

```diff
@@ -4,82 +4,108 @@
+You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 4.7.1:
+
+```shell
+ansible-galaxy collection install arista.avd:==4.7.1
+```
 
-### Custom plugins & modules
+See [using Ansible collections](https://docs.ansible.com/ansible/devel/collections_guide/index.html) for more details.
 
-See the section "Ansible Collection Plugins" on the menu to the left.
+### Additional Python Dependencies
 
-## Additional resources
+The AVD collection requires the installation of additional Python packages. To ensure you install the correct versions, run the following commands:
 
-- Ansible [EOS modules](https://docs.ansible.com/ansible/latest/collections/arista/eos/index.html) on Ansible documentation
-- Ansible [CloudVision modules](https://cvp.avd.sh/en/stable/)
-- [CloudVision Portal](https://www.arista.com/en/products/eos/eos-cloudvision)
-- [Arista Design and Deployment Guides](https://www.arista.com/en/solutions/design-guides)
+```shell
+export ARISTA_AVD_DIR=$(ansible-galaxy collection list arista.avd --format yaml | head -1 | cut -d: -f1)
+pip3 install -r ${ARISTA_AVD_DIR}/arista/avd/requirements.txt
+```
 
-## Ask a question
+### Enable jinja2 extensions
 
-Support for the `arista.avd` collection is provided by the community directly in this repository. If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions).
+In your `ansible.cfg` file, add the following modifications:
 
-### Official Arista support
+```shell
+jinja2_extensions=jinja2.ext.loopcontrols,jinja2.ext.do
+duplicate_dict_key=error
+```
+
+## Use Cases
+
+Please see the official documentation for examples in data center, campus, and WAN environments.
+
+- [Getting started](https://avd.arista.com/stable/docs/getting-started/intro-to-ansible-and-avd.html)
+- [Arista NetDevOps Examples](https://github.com/aristanetworks/netdevops-examples)
 
-AVD version 4.x releases with full support from Arista TAC. If your organization has the appropriate support agreement, please don't hesitate to contact TAC with any questions or issues.
+### Testing
+
+Please see our [pipeline workflows](https://github.com/aristanetworks/avd/actions) to view the in-depth testing performed against the collection.
 
 ## Contributing
 
-Contributing pull requests are gladly welcomed for this repository. If you are planning a big change, please start a discussion first to make sure we'll be able to merge it. Please see [contribution guide](https://avd.arista.com/stable/docs/contribution/overview.html) for additional details.
+Contributing pull requests are gladly welcomed for this repository. If you are planning a big change, please start a discussion first to make sure we'll be able to merge it. Please see [contribution guide](docs/contribution/overview.md) for additional details.
```

Comment:
I believe this one should use an absolute link, since it will be posted on galaxy.